index.html

```diff
@@ -103,6 +103,7 @@ <h4>Language/technology-specific</h4>
       <li><a href="http://nodegirls.io/">NodeGirls</a> - free one day workshops on Node JS.</li>
       <li><a href="http://www.meetup.com/PyLadiesLondon/">PyLadies</a> - informal sessions, workshops and study groups to learn Python through small projects, as well as talks (aiming to support more women and genderqueers to become active participants and leaders in the Python open-source community).</li>
       <li><a href="http://railsgirls.com/london">Rails Girls</a> - free 1.5 day workshops on web development with Ruby on Rails.</li>
+      <li><a href="https://www.meetup.com/en-AU/ReactJS-Girls-London"/>ReactJS Girls London</a> - for anyone interested in ReactJS and wanting to hear from female engineers and their experiences working with React.</li>
```

Comment:
URL should be `https://www.meetup.com/ReactJS-Girls-London` to not force locale to `en-AU`.